jwst/fits_generator/create_data.py

```diff
@@ -25,8 +25,11 @@ def get_proposals(base_directory='.'):
     proposal_list = []
     for dirname, subdirlist, filelist in os.walk(base_directory):
         for name in filelist:
-            if name.endswith('.prop'):
-                proposal_list.append((dirname, name))
+            if name.startswith('.'):
```

Comment:
I think this could be compacted down by using `not`:  

So either:
```
if not name.startswith('.'):
   if name.endswith('.prop'):
```
or 
```
if name.ednswith('.prop') and not name.startswith('.'):
```